src/avfoundation.cs

```diff
@@ -13009,7 +13498,7 @@ interface AVAssetDownloadStorageManagementPolicy : NSCopying, NSMutableCopying
 		NSDate ExpirationDate { get; [NotImplemented] set; }
 	}
 
-	[NoWatch, NoTV, NoMac, iOS (11,0)]
+	[NoWatch, NoTV, Mac (12,0), iOS (11,0)]
```

Comment:
catalyst ?

Reply:
same, it is the min version of catalyst.